backend-services/Services/UserSessionService.py

```diff
@@ -0,0 +1,46 @@
+from flask import Blueprint, jsonify
+from extensions import mongo
+
+
+user_session_service = Blueprint("user_session", __name__)
+
+
+@user_session_service .route("/get-all-sessions", methods=["GET"])
+def getAllSessions():
+    session_collection = mongo.db.sessions.find({})
+    sessions = []
+    index = 0
+    for session in session_collection:
+        clean_session = {
+          'id': index,
+          "user_id": session["user_id"],
+          "first_received_at": session["first_received_at"],
+          "topic": session["topic"],
+          "focus_score": session["focus_score"],
+          "transcribed_at": session["transcribed_at"],
+          "transcribed_speech": session["transcribed_speech"]
+        }
+        sessions.append(clean_session)
+        index += 1
+        all_sessions = {
+          "sessions": sessions
+        }
+    return jsonify(all_sessions)
+
+
+@user_session_service .route("/insert-session", methods=["POST"])
```

Comment:
It looks like this is not working properly. I am unable to hit this route in Postman
![image](https://user-images.githubusercontent.com/36316432/84740966-2aa06800-af63-11ea-82f7-a8768af9c2ae.png)

Reply:
The error was on my end, I didn't realize that this one needed parameters to be passed into this specific route. I am able to hit this route when passing in some parameters